Sources/Host/POSIX/PTrace.cpp

```diff
@@ -64,7 +64,13 @@ ErrorCode PTrace::attach(ProcessId pid) {
 
   DS2LOG(Debug, "attaching to pid %" PRIu64, (uint64_t)pid);
 
-  if (wrapPtrace(PTCMD(ATTACH), pid, nullptr, nullptr) < 0)
+#if defined(OS_LINUX) || defined(OS_FREEBSD)
+  auto cmd = PTCMD(ATTACH);
+#elif defined(OS_DARWIN)
+  auto cmd = PT_ATTACHEXC;
```

Comment:
This _could_ not be on all darwin version, "man ptrace" on my mac doesn't hace PT_ATTACHEXC. What version of mac did you tests ? I will tests my now.